packages/auth/e2e/auth.e2e.js

```diff
@@ -39,9 +39,18 @@ describe('auth()', () => {
   });
 
   describe('applyActionCode()', () => {
+    xit('works as expected', async () => {
```

Comment:
Is this still valid to be skipped?

Reply:
Yeah, this is a new test that I added. But have no way of testing it at the moment. Will be needed in the future for full coverage at least